

FAIL back() in a detached window assert_equals: expected 2 but got 1

